compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -93,6 +93,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val YdetailedStats = BooleanSetting("-Ydetailed-stats", "show detailed internal compiler stats (needs Stats.enabled to be set to true).")
   val Yheartbeat = BooleanSetting("-Ydetailed-stats", "show heartbeat stack trace of compiler operations (needs Stats.enabled to be set to true).")
   val Yprintpos = BooleanSetting("-Yprintpos", "show tree positions.")
+  val YprintposSyms = BooleanSetting("-Yprintpos-syms", "show symbol definitions positions.")
```

Comment:
Since `-Yprintpos` is about to change to `Yprint-pos`, I guess we also need an additional hypen here: `-Yprint-pos-syms`.

Reply:
Sure, in which PR is Yprintpos changed to Yprint-pos ?